src/components/structures/ScrollPanel.js

```diff
@@ -535,21 +539,29 @@ export default class ScrollPanel extends React.Component {
      * @param {object} ev the keyboard event
      */
     handleScrollKey = ev => {
+	let isScrolling = false;
```

Comment:
This block seems to be using tab indents instead of spaces somehow... 🤔

Reply:
thanks for the heads up, looks like my merge tool did not follow what `.editorconfig` instructed it